src/airport.js

```diff
@@ -1 +1,41 @@
+const airport = {
+    //const as in 1 airport so that wont change 
+    itemsInAirport: [],
+    defCapacity: 40,
 
+    maxCapacity(newCapacity) {
+        this.defCapacity = newCapacity;
+    },
+
+
+    itemLand: function (item) {
+        if (item.id) {
+            this.itemsInAirport.push(item);
+        }
+    },
+
+    isAirportFilled() {
+        return this.defCapacity >= 40;
+    },
+
+    itemDispatched: function (item) {
+        if (item?.id) {
+            const index = this.itemsInAirport.indexOf(item);
+            const remove = this.itemsInAirport.splice(index, 1);
```

Comment:
should have adda an if statement here to make my code a bit more flexible not sure tho (if (index !== -1)

Reply:
Next challenge i will try and focus on making my code cleaner